envs/6.2

```diff
@@ -0,0 +1,12 @@
+COMPOSE_PROJECT_NAME=polymesh
+
+CHAIN_IMAGE=polymeshassociation/polymesh:6.2.0-staging-debian
+REST_IMAGE=polymeshassociation/polymesh-rest-api:v4.1.0
+SUBQUERY_IMAGE=polymeshassociation/polymesh-subquery:v12.2.0
+PG_PORT=5432
+REST_PORT=3004
+SQ_PORT=3001
+VAULT_VERSION=latest
+VAULT_PORT=8200
+VAULT_REST_PORT=3014
+ENABLE_VAULT=0
```

Comment:
If enable vault is falsey, then there isn't a point to specifying the values imo

Reply:
I am not even sure where this is referenced tbh